pybossa/cache/projects.py

```diff
@@ -161,6 +161,18 @@ def n_tasks(project_id):
     return n_tasks
 
 
+@memoize(timeout=timeouts.get('APP_TIMEOUT'), cache_group_keys=[[0]])
+def n_tasks_not_gold(project_id):
+    """Return number of tasks of a project."""
```

Comment:
doc string can be updated to include that tasks returned would be w/o gold tasks similar to func name.